.travis.yml

```diff
@@ -12,7 +12,17 @@ env:
   - TOXENV=py33-1.8
   - TOXENV=py34-1.8
   - TOXENV=py34-1.9
+  - TOXENV=py34-1.10
+  - TOXENV=py34-1.11
   - TOXENV=py34-master
+  - TOXENV=py35-1.9
+  - TOXENV=py35-1.10
+  - TOXENV=py35-1.11
```

Comment:
Update: but the build fails; https://travis-ci.org/charettes/django-colorful/jobs/220687624 let me check what's going on